programs/protocol/src/data/offer.rs

```diff
@@ -0,0 +1,34 @@
+use anchor_lang::prelude::*;
+
+use crate::ShipmentData;
+
+#[zero_copy]
+#[derive(Debug, Default, PartialEq, Eq)]
+pub struct OfferDetails {
+    pub payment: u64,
+    pub collateral: u64,
```

Comment:
what's the difference?

Reply:
payment is whatever the truct driver will get for the transport
collateral is how much he will have to stake as collateral for shipment value